runtime/cudaq/platform/default/rest/helpers/quantinuum/quantinuum.config

```diff
@@ -16,7 +16,7 @@ GEN_TARGET_BACKEND=true
 LINKLIBS="${LINKLIBS} -lcudaq-rest-qpu"
 
 # Define the lowering pipeline, here we lower to Base QIR
-PLATFORM_LOWERING_CONFIG="expand-measurements,canonicalize,cc-loop-unroll{full-unroll=true},canonicalize,func.func(lower-to-cfg),canonicalize,quantinuum-gate-set-mapping"
+PLATFORM_LOWERING_CONFIG="expand-measurements,canonicalize,cc-loop-unroll{full-unroll=true},canonicalize,func.func(lower-to-cfg),canonicalize,func.func(quake-multicontrol-decomposition),quantinuum-gate-set-mapping"
```

Comment:
should this be default for all remote targets? If so, should we add it (and maybe some of the others before this) as a member string on RemoteRESTQPU. Could be something we think about in a refactor after the release.

Reply:
After the release, we need to rethink how pipelines are built. At this points they seem to be scattered through compiler code, runtime code, and the driver script.

Reply:
We could certainly make more/better use of prefabbed named pass pipelines. Certain collections of passes should be run together in certain orders.

The driver is the right place to build pipelines in general. That's a central "reason to exist" for a driver, which translates the combinatorics of command-line options into compiler actions. The runtime needs to assemble a pipeline as well because of late compilation, supporting Python, etc. Still using named pipelines and maybe a common "config" file could get us closer to a SPOT ideal. The early and late compilation steps aren't going to be completely identical because they're serving different purposes.